kotlin/src/main/com/looker/rtl/Transport.kt

```diff
@@ -266,38 +228,29 @@ class Transport(val options: TransportOptions) {
         } + addQueryParams(path, queryParams)
     }
 
-    inline fun <reified T> request(
+    inline fun <reified TSuccess, reified TFailure> request(
         method: HttpMethod,
         path: String,
         queryParams: Values = mapOf(),
         body: Any? = null,
         noinline authenticator: Authenticator? = null
-    ): SDKResponse {
-        // TODO get overrides parameter to work without causing compilation errors in UserSession
```

Comment:
Do I need to preserve these TODOs?

Reply:
We need to functionality that TODO was for, but not necessarily in that place